example_github_workflow_files/TagBot.yml

```diff
@@ -0,0 +1,19 @@
+name: TagBot
+on:
+  schedule:
+    - cron: 0 0 * * *
+  pull_request:
+    types:
+      - closed
+jobs:
+  TagBot:
+    if: github.event_name == 'schedule' || github.event.pull_request.merged
+    runs-on: ubuntu-latest
+    steps:
+      - uses: julia-actions/setup-julia@v1
+        with:
+          versions: '1'
```

Comment:
Julia 1.5.2 is included by default in the hosted runners, there should be no need to install it separately anymore unless TagBot needs a specific version.